### DISCUSSION TITLE
Chrome Extension

### DISCUSSION THREAD

**Discussion by GeErghon** [dlemstra/magick-wasm#187] [Q&A]
Hi,
I'm trying to use magick.wasm in a Chrome extension, but every time I call the initializeImageMagick method, I get the error: "CompileError: WebAssembly.instantiateStreaming(): Refused to compile or instantiate WebAssembly module because neither 'wasm-eval' nor 'unsafe-eval' is an allowed source of script in the following Content Security Policy directive: 'script-src 'self'."
I would greatly appreciate any guidance on the right path to take.

**Comment by dlemstra** [comment]
I don't have any experience with chrome extensions so I cannot really help you with this. How are you adding `magick.wasm` to your chrome extension?

**Reply by GeErghon** [reply]
I'm using it like this 
import { ImageMagick, initializeImageMagick } from './magick-wasm/index.js'

thanks

**Reply by dlemstra** [reply]
I meant where you load the `.wasm` file from?

**Reply by GeErghon** [reply]
it is added to the extension package and available like a normal file in a html page with a relative uri

**Reply by kaya51** [reply]
Can you share a minimal, reproducible example extension?
